geode/license.md

```diff
@@ -0,0 +1 @@
+https://github.com/apache/geode/blob/develop/LICENSE
```

Comment:
This gets embedded directly in the image description as-is -- I'd recommend looking at the `license.md` files of other images for ideas of common formattings for this file.